Files/UserControls/Widgets/FolderWidget.xaml.cs

```diff
@@ -140,11 +140,14 @@ public void Dispose()
 
         private async Task GetItemsAddedIcon()
         {
-            foreach (var item in ItemsAdded.ToList())
+            for (int i = 0; i < ItemsAdded.Count; i++)
```

Comment:
The ToList() was added to resolve 2267876499u in 1.9.30. 2267876499u is from 1.9.28, unless the error pops out in 1.9.30 and later I'd leave this as is.

Reply:
Reverted and added try-catch